addons/overpressure/CfgEventHandlers.hpp

```diff
@@ -14,7 +14,7 @@ class Extended_PostInit_EventHandlers {
 class Extended_FiredBIS_EventHandlers {
     class CAManBase {
         class ADDON {
-            firedBIS = QUOTE(if (local (_this select 0)) then {_this call FUNC(firedEHBB);};);
+            clientFiredBIS = QUOTE(_this call FUNC(firedEHBB););
```

Comment:
I switched the `if !([_firer] call EFUNC(common,isPlayer)) exitWith {};` in fireLauncherBackblast
to `_firer != ACE_player` in firedEHBB  
(because local + isPlayer could only ever be `ace_player`... I think?)

In testing re-adding the local check in the EH is slightly faster than always doing the params and `_firer != ACE_player` check when non players are shooting, so I will leave as it was.